applications/plot_layout_array.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application plots the layout array and saves into pdf and png files.
+
+    It accepts as input the telescope list file, the name of the layout, a sequence of arguments
+    with the telescope files or a sequence of arguments with the layout names.
+
+    A rotation angle in degrees can be passed in case the array should be rotated before plotting.
+    A sequence of arguments for the rotation angle is also permitted, in which case all of them
+    are plotted and saved separately.
+
+
+    Command line arguments
+    ----------------------
+    figure_name (str, optional)
+        File name for the pdf output (without extension).
+    telescope_list (str, optional)
+        The telescopes file (.ecsv) with the array information.
+    layout_array_name (str, optional)
+        Name of the layout array.
+    rotate_angle (float, optional)
+        Angle to rotate the array before plotting (in degrees).
+    show_tel_label (bool, optional)
+        Shows the telescope labels in the plot.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    .. code-block:: console
+
+        python applications/plot_layout_array.py --figure_name northern_array_alpha \
+        --layout_array_name test_layout
```

Comment:
I modified this to be a working example

Reply:
I dont understand. What would be a working example? And have you pushed any changes to the PR ? I don't see your commit

Reply:
Sorry, I committed but forgot to push...
See now the changes.